core-plugins/widgets/File-batchsource.json

```diff
@@ -28,6 +28,44 @@
           "name": "format",
           "widget-attributes": {
             "plugin-type": "validatingInputFormat"
+          }
+        },
+        {
+          "widget-type": "number",
+          "label": "Sample Size",
+          "name": "sampleSize",
+          "widget-attributes": {
+            "default": "1000",
+            "minimum": "1"
+          }
+        },
+        {
+          "widget-type": "keyvalue-dropdown",
+          "label": "Override",
+          "name": "override",
+          "widget-attributes": {
+            "key-placeholder": "Field Name",
+            "value-placeholder": "Data Type",
+            "dropdownOptions": [
+              "boolean",
+              "bytes",
+              "double",
+              "float",
+              "int",
+              "long",
+              "string",
+              "date",
```

Comment:
Are date and time types supported?

Reply:
`Date` (eg., `2021-12T12:34`, `2021-05-19`, ... ) and `Time` (e.g., `22:30`, `23:59:59Z`,...) values of ISO-806 standard can be detected correctly as of `Date` and `Time` data types respectively. 

Even though the output schema shows `Date` data type, when the source plugin (e.g., File Source) reads such coolumns in the background,  it will consider them as Strings.

Reply:
Yes, let's remove date and time if the data is just as strings.